modules/ROOT/pages/manage-audit-logs.adoc

```diff
@@ -0,0 +1,292 @@
+Setting enabled to true without any specified enabled events will only enable events with `DefaultEnabled` set to `true`. 
+To view the list of audit events, see xref:audit-log-events.adoc[]
+
+=== Configuration with the Database Config
+
+You can alter the configuration per Sync Gateway database by changing the database config.
+
+Below is an example of a simple database configuration.
+
+[source, json]
+----
+{
+  "bucket": "todo", 
+  "num_index_replicas": 0,
+  "logging": {
+               "audit": {
+                          "enabled": false,
+                          "enabled_events": [],
+                          "disabled_users": [
+                                              {
+                                                "name": "user1",
+                                                "domain": "sgw"
+                                              }
+                                            ]
+                        },
+  "disabled_roles": [
+                      {
+                        "name": "role1",
+                        "domain": "sgw"
+                      }
+                    ]
+               }              
+}  
+----
+
+For further database configuration options, see xref:configuration-schema-database.adoc[].
+
+IMPORTANT: If you enable audit logging in your bootstrap config it is enabled globally.
+Conversely, it is important to note that if audit logging is enabled in your database config but *not* your bootstrap config, no audit logging will occur.
+
+Couchbase recommends you only use configuration at the database level if you wish to alter your settings per database. 
+
+== Filtering Audit Logs
+
+You can filter audit logs to customize events that are stored in the log file. 
+You can alter the following to affect events stored in the log file:
+
+* Enabling and disabling users - This affects all events related to a specified user.
+If a given user is disabled, no audit events from the user will be logged.
+
+* Enabling and disabling events - This affects the specified events globally.
+If a given event is disabled, it will not be logged regardless of the user.
+
+NOTE: If an audit event has `DefaultEnabled` set to `true`, it does not need to be explicitly listed in the `enabled_events: []` array.
+See xref:audit-log-events.adoc[] for more information on specific audit events.
+
+=== Enabled and Disabled Events
+
+Events can be enabled per database or globally.
+
+The example below shows a section of a global bootstrap config which would enable Public HTTP API requests and Public API user authenticated events only.
+
+[source, json]
+----
+"logging": {
+             "audit": {
+                        “audit_log_file_path”: “/path/to/file.log”,
+                        “enabled”: true,
+                        “enabled_events”: [53270, 53280]
+                      }
+            }
+----
+
+Enabling the events at runtime without modifying the entire database
```

Comment:
At the end of the day these endpoints are going to modify the database (by updating the audit config for the database) - so this description isn't exactly right.  It would be better described as something like 'Modifying the database audit logging config independently'